website/content/docs/operators/deployment/schedulers/dcos.md

```diff
@@ -98,11 +99,8 @@ heron.marathon.scheduler.uri: "<core.dcos_url>/service/marathon"
 # Invoke the IScheduler as a library directly
 heron.scheduler.is.service:                  False
 
-# location of the core package
-heron.package.core.uri:                      https://github.com/twitter/heron/releases/download/0.14.7/heron-core-0.14.7-ubuntu.tar.gz
-
-# docker repo for executor
-heron.executor.docker.image: 'ndustrialio/heron-executor:jre8'
+# docker repo for heron with core packages installed
+heron.executor.docker.image: 'streamlio/heron:latest-ubunut14.04'
```

Comment:
With the tag, the only comment i have is to be sure we have tags corresponding to versions number. So, ideally, you'd be referencing something like `streamlio/heron:0.14.8`